src/main/io/gps.h

```diff
@@ -114,12 +114,19 @@ typedef struct gpsLocation_s {
     int32_t altCm;                  // altitude in 0.01m
 } gpsLocation_t;
 
+/* Accuracy of position estimation = device accuracy * DOP */
+typedef struct gpsDilution_s {
+    uint16_t pdop;                  // positional DOP - 3D (* 100)
+    uint16_t hdop;                  // horizontal DOP - 2D (* 100)
+    uint16_t vdop;                  // vertical DOP   - 1D (* 100)
+} gpsDilution_t;
+
```

Comment:
It might be helpful, either in these comments, or in later comments after scaling, to describe the units of each of `prop` etc?    After *100, are these values are effectively the radius of precision in cm?

Reply:
Ah yes I recall.  It's a modifier of the specified accuracy of the device used.  Thanks.

Reply:
So no matter how many sattellites are locked, as long as DOP <= 1 we can be pretty confident about having a trustworthy GPS signal. For values >1 we should get a bit nervous. Values >3 are very poor performance, imo, considering we are aiming for very high precision.